lib/src/testing/test-runner.spec.ts

```diff
@@ -176,6 +176,26 @@ describe("test runner", () => {
     expect(result).toBe(true);
   });
 
+  test("include draft", async () => {
+    const contract = parseAndCleanse(
+      `${testExamplesBasePath}/draft-endpoint.ts`
+    );
+
+    const scopeDraft = nock(baseUrl)
+      .post("/state/initialize")
+      .reply(200)
+      .post("/companies", { name: "My Company", private: true })
+      .reply(201, { name: "My Company" })
+      .post("/state/teardown")
+      .reply(200);
+
+    const result = await testRunner.test(contract, {
+      includeDraft: true
+    });
+    expect(scopeDraft.isDone()).toBe(true);
+    expect(result).toBe(true);
```

Comment:
Would it be worth having another test where it fails, and confirming that it's true when includeDraft is false, and false when includeDraft is true?

Reply:
The check here is line 195 - `expect(scopeDraft.isDone()).toBe(true);`.
There is an [existing test](https://github.com/airtasker/spot/blob/test-runner-include-draft-flag/lib/src/testing/test-runner.spec.ts#L150) that checks that without `includeDraft` draft tests are excluded `expect(scopeDraft.isDone()).toBe(false);`

Reply:
Added